src/main/resources/static/css/style.css

```diff
@@ -93,6 +93,7 @@ a {
 	background-color: #4c4c4d;
 	color: #FFFFFF;
 	border-bottom: 8px solid #383839;
+	overflow: auto;
```

Comment:
![Screenshot 2020-07-01 01 04 27](https://user-images.githubusercontent.com/60111772/86296014-19f52080-bbf8-11ea-92f0-17a5541524eb.png)

Reply:
I wonder, is it possible to center the contents on small screens? Imo would look a little better.

Reply:
Gonna give a try, nice idea.

Reply:
![image](https://user-images.githubusercontent.com/60111772/87251121-a3fb7f80-c469-11ea-815d-4bfbd48b1ab7.png)

It looks like this with the new commit. I bravely hope, people do not use mobile phones with less than 320px horizontal resolution.